source/deployment/advanced-permissions.rst

```diff
@@ -56,17 +56,28 @@ The interface for naming, assigning teams, and editing permissions in a Team Ove
 
 .. image:: ../images/team-scheme.png
 
-Channel Override Permissions (E20)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Channel moderation (Beta) (E20)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-*Available in a future release of Enterprise Edition E20*
+*Available in Enterprise Edition E20*
 
-Allows Admins to restrict permissions within specific channels. Permissions under consideration for this phase include:
+From v5.22, the channel moderation settings allow Admins to restrict actions within specific channels. These actions include: 
 
 - **Read-only Channels:** The ability for Admins to turn off posting in specified channels.
+- **Restrict Post Reactions:** Turn off the ability for members and guests to post reactions.
 - **Restrict Channel Mentions:** Turn off the ability for users to post channel wide mentions (@-all/channel/here) in specified channels.
 - **Channel member management:** Restricting adding and removing channel members to Admins only in specified channels.
 
+These settings are modified in **System Console > User Management > Channels**. 
+
+**Note:**
+These settings are applicable only to Guests and Members. System, Team, and Channel Admins are not affected. 
+
+The availability of channel moderation settings can also be affected by existing system and team permissions configurations. If there are existing configurations that override the channel settings you would like to apply, it will be indicated in the UI. These settings can then be adjusted in the relevant panel in the **Permissions** section of the System Console. 
+
+This feature includes the ability to set channel moderation settings in the **Team Override Scheme** so Public and Private channels have individual moderation settings. In some cases, permissions applied in the **Team Override Scheme** take precedence and affect the availability of certain settings. This is indicated in the **Channel Moderation (Beta)** UI where applicable.  
```

Comment:
I am not sure what is meant to be communicated here. The channel specific schemes (that underly the channel moderation settings) cannot be set or modified from the permissions page. Only the team level permissions (which do constrain channel schemes). Also, from the perspective of public/private, there are no differences wrt channel moderation settings - at least for this release.

Reply:
Thanks for clarifying. So, re-reading this, I'd say that the first paragraph is correct (based on what I can see in the UI and your comments). But, paragraph two is most likely a confusion based on the test feedback in the two screenshots below. 

Happy to remove it if it's confusing/extraneous. In hindsight, the UI provides enough guidance that we don't really need to document this - thoughts? 

<img width="758" alt="Screen Shot 2020-04-07 at 18 03 15" src="https://user-images.githubusercontent.com/4223530/78692396-1d7b5300-78fa-11ea-8356-145f452903ab.png">

<img width="731" alt="Screen Shot 2020-04-07 at 18 02 54" src="https://user-images.githubusercontent.com/4223530/78692395-1c4a2600-78fa-11ea-9187-9ec0d352458b.png">

Reply:
I removed the section around public and private channel settings because it is more back-end, expected behavior and was extraneous/confusing to mention it. Thanks for picking that up!